OpenTESArena/src/Entities/Doodad.h

```diff
@@ -12,8 +12,8 @@
 class Doodad : public Entity
 {
 private:
-	Animation animation;
 	Double3 position;
+	Animation animation;
```

Comment:
I put the declaration back where it was and instead changed the order of initialization to match.